pubpub/platform
# fix: make CORS work for site route

## Issue(s) Resolved

Could not use `/site` api routes client side due to CORS errors.

## Test Plan

1. In `croccroc`, create an API key which is able to `GET` pubs at the very least
2. Run the following query in the console of your browser on any page that isn't `localhost` (e.g. here on github), substituting `<key>` obviously.
```js
await (await fetch(`http://localhost:3000/api/v0/c/croccroc/site/pubs`, {
    headers: {
        Authorization: `Bearer <key>`
    }
})).json()
```
3. Observe a response


## Screenshots (if applicable)

## Optional

### Notes/Context/Gotchas

### Supporting Docs
